src/main/java/sjchat/users/UserServiceServer.java

```diff
@@ -0,0 +1,89 @@
+package sjchat.users;
+
+import java.io.IOException;
+import java.util.Random;
+
```

Comment:
nit: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

In general, formatting according to the style guidelines (Google's)